usaspending_api/recipient/models.py

```diff
@@ -81,7 +81,7 @@ class RecipientProfile(models.Model):
     recipient_unique_id = models.TextField(null=True, db_index=True)
     recipient_name = models.TextField(null=True, db_index=True)
     recipient_affiliations = ArrayField(base_field=models.TextField(), default=list, size=None)
-    award_types = ArrayField(base_field=models.TextField(), default=list, size=None)
+    award_types = ArrayField(base_field=models.TextField(), default=list, size=None, db_index=True)
```

Comment:
Unsure why, but the current index is a GIN index. Most likely need to add index definition in the Meta class